src/fetchWithGraphQl.ts

```diff
@@ -12,7 +12,7 @@ function fetchWithGraphQl(query: GraphQlQuery, token: string | null = null): Pro
       ...authHeaders,
       Accept: 'application/json',
       'Content-Type': 'application/json',
-      Store: 'default',
+      Store: store,
```

Comment:
Don't you need to provide a default value here?

Reply:
Good point, I will address this in a new commit